siliconcompiler/core.py

```diff
@@ -169,6 +169,10 @@ def _load_module(self, module_name, raise_error=False):
 
         return None
 
+    ###########################################################################
+    def _get_loaded_modules(self):
```

Comment:
What is the benefit of this method?
Is `chip.modules` supposed to be private?
Also are they definitely "loaded" at any point when calling it?

Reply:
At some point we need to go through and make these things private.